.github/container/levanter-cache-warn.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+YELLOW='\033[0;33m'
+NOCOLOR='\033[0m'
+
+echo -e "${YELLOW}
+WARNING: To speed up the model training it is recommended to cache the datasets with:
```

Comment:
so it's not so much about speeding up (in a "real" run it shouldn't really impact throughput much at all aside from initial warmup). It does make a material difference in unit tests.

 I'll make a more detailed help section and maybe I can PR it to this section?

Reply:
@dlwh Sounds like a plan

Reply:
> I'll make a more detailed help section and maybe I can PR it to this section?

No objections from my side. 
@yhtang What do you think about it?